src/features/surveys/components/SurveySuborgsCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { ChangeEvent } from 'react';
+import messageIds from '../l10n/messageIds';
+import SurveyDataModel from '../models/SurveyDataModel';
+import { useMessages } from 'core/i18n';
+import useModel from 'core/useModel';
+import ZUICard from 'zui/ZUICard';
+
+import { Box, Switch } from '@mui/material';
```

Comment:
I usually group imports into two groups. The first containing external dependencies (React, MUI etc) and the second containing internal dependencies (code that we wrote ourselves). Some don't care about the grouping, which I think is fine.

In this case you have two groups, but the only logic separating them seems to be "MUI in one group, everything else in another".

Reply:
I like the two groups division! But how we can separate them if the lint criteria is first single imports and then multiple imports?

Reply:
I'm not sure I understand the question. The order would be this:

1. Single imports from external libraries
2. Multiple imports from external libraries
3. Blank line (group separator)
4. Single imports from within Zetkin
5. Multiple imports from within Zetkin

In the case of this file, it would be:

```tsx
import { ChangeEvent } from 'react';
import { Box, Switch } from '@mui/material';

import messageIds from '../l10n/messageIds';
import SurveyDataModel from '../models/SurveyDataModel';
import { useMessages } from 'core/i18n';
import useModel from 'core/useModel';
import ZUICard from 'zui/ZUICard';
```